templates/patient/card/appointments.html.twig

```diff
@@ -0,0 +1,100 @@
+{% extends "patient/card/card_base.html.twig" %}
+
+{% macro appointmentDetail(appointment) %}
+    {% set a = appointment %}
+    <div class="list-group-item p-1" style="background-color: {{ a.bgColor }};">
+        <div class="d-flex w-100 justify-content-between">
+            <h6>
+                {% if a.pc_catid not in therapyGroupCategories %}
+                <a href="#" onclick="return oldEvt({{ a.jsEvent }})">
+                    {% endif %}
+                    {{ a.pc_catname|xlt }}
+                    {% if a.pc_hometext %}&nbsp;<i class="far fa-fw fa-comment-dots"></i>{% endif %}
+                    {% if a.pc_recurrtype %}&nbsp;<i class="fa fa-retweet"></i>{% endif %}
+                    {% if a.pc_catid not in therapyGroupCategories %}
+                </a>
+                {% endif %}
+            </h6>
+            <small class="text-right">
+                {{ a.dayName|xl }},&nbsp;
+                {{ a.pc_eventDate|text|shortDate }}
+                {{ a.pc_eventTime }}&nbsp;{{ a.displayMeridiem|xlt }}
+            </small>
+        </div>
+        <div class="m-0 d-flex w-100 justify-content-between text-muted">
+            <span>{{ a.uname }}</span>
+            <span>{{ a.pc_status }}
+                {% if a.pc_catid in therapyGroupCategories %}
+                {{ "Group name"|xlt }}: {{ a.groupName|text }}
+                {% endif %}
+            </span>
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro additionalAppointment(appointmentDate) %}
+    <div class="list-group-item p-1">
+        <div class="m-0 pt-2 d-block w-100 text-muted text-center">
+            {{ appointmentDate|text }}&nbsp;(+)
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro sectionHeader(title, iconClasses) %}
+    <div class="list-group-item pl-0 pb-1 border-bottom-0">
+        <div class="text-dark font-weight-bolder">
+            <i class="fa fa-fw {{ iconClasses }}"></i>&nbsp;{{ title|xlt }}
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro noItems(text) %}
+    <div class="list-group-item text-center">
+        {{ text|xlt }}
```

Comment:
as discussed above, better to translate the constant directly if possible and then only escape it here (looks like the constants are below)